cmd/helm/install.go

```diff
@@ -154,7 +154,11 @@ func newInstallCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 				return errors.Wrap(err, "INSTALLATION FAILED")
 			}
 
+<<<<<<< HEAD
 			return outfmt.Write(out, &statusPrinter{rel, settings.Debug, false, false})
+=======
+			return outfmt.Write(out, &statusPrinter{rel, settings.Debug, false, client.HideNotes})
+>>>>>>> 5cc99712 (Added support for hiding notes from install/upgrade output)
```

Comment:
Oops, you included the merge conflicts.

Reply:
The problem is that there are additional conflicts that aren't being picked up, where both branches have `&statusPrinter{rel, settings.Debug, false, false}`, so they all need to be amended. I'll keep working on this. I've preserved the original branch as `old-chart-notes` in my fork. The current branch is free of merge conflicts but will not build.

Reply:
@joejulian finally got it, all of the changes should be reconciled

Reply:
Thank you